Makefile

```diff
@@ -90,7 +95,7 @@ test: export CXXFLAGS := --this_flag_does_not_exist # Env removed in text_gcovr.
 test: export GCOV := $(GCOV)
 
 test:
-	cd gcovr/tests && make clean
+	cd gcovr/tests && make clean > /dev/null
```

Comment:
Or `make --quiet`? But the /dev/null Unix-ism seems to work on the Windows-based tests as well, so that seems fine as well.

Reply:
There is no option in make to suppress the output of the targets.

Reply:
`make -s` definitely works to suppress the usual output, and is even Posix-compliant. The `--quiet` or `--silent` options are GNU extensions to Make.

Reply:
I overlooked the `--quit` in the help output. I'll change this.